build.sbt

```diff
@@ -497,7 +498,7 @@ lazy val scalafix = tlScalafixProject
   )
   .outputConfigure(_.dependsOn(oteljava).disablePlugins(ScalafixPlugin))
   .testsSettings(
-    crossScalaVersions := Seq(Scala213),
+    crossScalaVersions := Seq(Scala212),
```

Comment:
@armanbilge I have a feeling I'm missing something.

The `scalafix-rule` must be compiled with Scala 2.12, so the sbt can use it.

`scalafix-input`, `scalafix-output`, and `scalafix-tests` must use Scala 2.13 and Scala 3 targets, right?

Reply:
> The `scalafix-rule` must be compiled with Scala 2.12

It doesn't have to be. You can compile it with 2.13 as well. Then the user must set `scalafixScalaBinaryVersion` to `2.13`.

> so the sbt can use it

It's not related to sbt.

---

In any case, here's how I set it up in Feral. It looks like I build the rule with 2.12 and test it with 2.13. I probably didn't bother testing with 3 because it makes the build too complicated.

https://github.com/typelevel/feral/blob/fa51e2f5aa7a082536a4a71e5375343a7c2d72f4/build.sbt#L206-L226

Reply:
The configuration of the scalafix projects is identical to Feral's in this PR. So we should be good, I guess.